docs/elements/elements.html

```diff
@@ -10,6 +10,11 @@
   - /assets/packages/@rhds/elements/elements/rh-subnav/rh-subnav-lightdom.css
 eleventyComputed:
   title: "{{ doc.pageTitle }} | {{ doc.slug | deslugify }}"
+  importElements:
+    - rh-subnav
+    - rh-cta
```

Comment:
If elements.html uses `layout-base.njk` which also  importsElements `rh-cta` and `rh-footer` do you need to repeat those here?  Assuming this is an override method not a incremental add.  Would the argument to do it this way be so you don't have to have to mental load of the full layout stack and which layouts import which elements?

Reply:
we can dedupe by routing through a `new Set()`, but ultimately

```
import './a.js';
import './a.js';
```
the browser will dedupe for you, so there's no correctness problem here ifaict.